sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -784,14 +778,15 @@ def test_metadata_is_routed_correctly_to_scorer(metaestimator):
         method = getattr(instance, method_name)
         method_kwargs = {"sample_weight": sample_weight}
         if "fit" not in method_name:
-            instance.fit(X, y)
+            instance.fit(X, y, **method_kwargs)
```

Comment:
Considering that we are now specifically checking the params and kwargs for specific methods, why do we need to send in these to fit when fit is not the method under consideration? Is it because some estimator needs this for correctness?

Reply:
That required to properly set request values, done now.

Reply:
Thanks!